### ISSUE TITLE
runtime error: index out of range

### ISSUE THREAD

**Issue by taohex** [mkchoi212/fac#18]
I installed fac from brew on macOS. Runtime error every time.

```
panic: runtime error: index out of range

goroutine 1 [running]:
main.main()
	/Users/mikechoi/src/gocode/src/github.com/mkchoi212/fac/main.go:117 +0x67f
```

**Comment by mkchoi212** [comment]
Can I ask you what the file with conflict markers look like?

**Comment by mkchoi212** [comment]
@taohex ???

**Comment by taohex** [comment]
_posts/2013-11-15-macos.md:751: new blank line at EOF.
(END)

**Comment by mkchoi212** [comment]
Thanks!

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by mkchoi212** [comment]
Fixed with 3114cf93bba24f1c269c5c58da53e902de9b5245

**Comment by taohex** [comment]
Thank you!
